exec-tests

```diff
@@ -69,7 +69,7 @@ function run_legacy {
     shift
 
     printf -- "\n\n\nRunning %s/%s legacy unit tests\n\n" "${_major}" "${_subtst}"
-    _time ${_major}/${_subtst}/unittests ${@}
+    COLUMNS=80 _time ${_major}/${_subtst}/unittests ${@}
```

Comment:
Oooh!  🎉

Reply:
Yeah, that was easier than I'd feared it might be.